frontend/src/modules/projects/index.js

```diff
@@ -63,16 +61,14 @@ define(function(require) {
   });
 
   Origin.on('dashboard:loaded', function (options) {
-    switch (options.type) {
-      case 'shared':
-        Origin.trigger('location:title:update', {title: 'Dashboard - viewing shared courses'});
-        Origin.contentPane.setView(ProjectsView, { collection: new SharedProjectCollection });
-        break;
-      case 'all':
-        Origin.trigger('location:title:update', {title: 'Dashboard - viewing my courses'});
-        Origin.contentPane.setView(ProjectsView, { collection: new MyProjectCollection });
-      default:
-        break;
+    if(options.type === 'shared') {
+      Origin.trigger('location:title:update', { breadcrumbs: ['dashboard'], title: Origin.l10n.t(langKey) });
```

Comment:
Not sure where `langkey` has come from?

Is this meant to be `'app.sharedprojects'` and `'app.myprojects'` respectively?